docs/develop/typescript/index.mdx

```diff
@@ -15,8 +17,7 @@ import * as Components from '@site/src/components';
 
 ![TypeScript SDK Banner](/img/assets/banner-typescript-temporal.png)
 
-:::info TYPESCRIPT SPECIFIC RESOURCES
-Build Temporal Applications with the TypeScript SDK.
+:::info TYPESCRIPT SPECIFIC RESOURCES Build Temporal Applications with the TypeScript SDK.
```

Comment:
Do we need the `Build Temporal Applications with the TypeScript SDK.` part here?